docs/en/operation/Tiered-Locality.md

```diff
@@ -93,18 +91,43 @@ If other locality tiers are left unset, they will not be used to inform locality
 
 ### Setting locality
 
-#### Locality script
+#### Option A: shell script
 
 By default, Alluxio clients and servers search the classpath for a script named
-`alluxio-locality.sh`.
-Output format of this script is a comma-separated list of tierName=tierValue pairs.
-The script name can be overridden by setting:
+`alluxio-locality.sh` (can be overridden by setting `alluxio.locality.script`).
```

Comment:
people don't know what this setting is yet at this point. recommend mentioning this in the next section when you first define what `alluxio.locality.script` is

Reply:
done